cloud_governance/cloud_resource_orchestration/clouds/azure/resource_groups/collect_cro_reports.py

```diff
@@ -0,0 +1,208 @@
+import logging
+from datetime import datetime, timedelta
+
+import typeguard
+
+from cloud_governance.cloud_resource_orchestration.clouds.azure.resource_groups.cost_over_usage import CostOverUsage
+from cloud_governance.cloud_resource_orchestration.clouds.common.abstract_collect_cro_reports import \
+    AbstractCollectCROReports
+from cloud_governance.common.logger.logger_time_stamp import logger_time_stamp
+from cloud_governance.main.environment_variables import environment_variables
+
+
+class CollectCROReports(AbstractCollectCROReports):
```

Comment:
Same question, why we need separate CRO class for Azure ?

Reply:
yes, the interface is same